content/community/members/anaconda/contents.lr

```diff
@@ -0,0 +1,18 @@
+join_date: 2022-03-21
+---
+level: gold
+---
+name: Anaconda
+---
+url: https://www.anaconda.com/
+---
+description:
+
+With more than 45 million users, `Anaconda <https://anaconda.com>`__ is the most popular operating system for AI providing access to the foundational open-source Python packages used in modern AI, data science and machine learning through a seamless platform. Anaconda pioneered the use of Python for data science, championed its vibrant community, and continues to steward open-source projects that make tomorrow’s innovations possible. Their enterprise-grade solutions enable corporate, research, and academic institutions around the world to harness the power of open-source for competitive advantage, groundbreaking research, and a better world.
```

Comment:
As a banner at the top of anaconda.com, it's reasonably obvious that "operating system" is used figuratively. But in this context, it would be much easier to take literally, and then people will think we don't even know what an operating system is.